### PR TITLE
adding live code-coverage for just python39

### PR DESCRIPTION
**Pull request by seankane-msft**
Breaking PR #15577 into two PRs, the latter will focus on the report generation. This PR is just for enabling code coverage

### CODE REVIEW

**Review by scbedd — DISMISSED** [review]

**Comment by scbedd** [comment]
@seankane-msft Can you do me a favor and do a quick search for any other `tests.yml` that invoke a custom matrix of platforms? Extend those with `coverageArg` as well.

**Comment by seankane-msft** [comment]
Addresses #15132

**Review by lmazuel — APPROVED** [review]
Approving the intent, need a engsys approval before merging. Thanks!

File: sdk/schemaregistry/tests.yml
Comment (scbedd) [inline sdk/schemaregistry/tests.yml:35]: We want one to remain right?

File: sdk/monitor/tests.yml
Comment (scbedd) [inline sdk/monitor/tests.yml:27]:
```suggestion
          CoverageArg: ''
```

File: sdk/formrecognizer/tests.yml
Comment (scbedd) [inline sdk/formrecognizer/tests.yml:47]:
```suggestion
            CoverageArg: ''
```

File: sdk/eventhub/tests.yml
Comment (scbedd) [inline sdk/eventhub/tests.yml:24]:
```suggestion
          CoverageArg: ''
```

Comment (scbedd) [reply sdk/eventhub/tests.yml:24]: We can chase down owners, but in the meantime, let's get it publishing coverage in general.

File: sdk/schemaregistry/tests.yml
Comment (scbedd) [inline sdk/schemaregistry/tests.yml:35]:
```suggestion
          CoverageArg: ''
```

**Review by scbedd — APPROVED** [review]
Please accept suggestions, otherwise g2g.

**Pull request merged**
